db/seeds.rb

```diff
@@ -5,3 +5,29 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+User.create(name: 'Example User',
+            email: 'example@railstutorial.org',
+            password: 'foobar')
+
+99.times do |n|
+    name  = Faker::Name.name
+    email = "example-#{n+1}@railstutorial.org"
+    password = "password"
+    User.create!(name: name,
+    email: email,
+    password:password)
+end
+
+users = User.all
+user  = users.first
+following = users[2..50]
+followers = users[3..40]
+following.each { |followed| user.follow(followed) }
+followers.each { |follower| follower.follow(user) }
+
+
+user  = users.last
+following = users[2..20]
+followers = users[3..10]
+following.each { |followed| user.follow(followed) }
+followers.each { |follower| follower.follow(user) }
```

Comment:
You need to add the node_module folder to gitignore file, we don't need these files!!!

Reply:
I added the line /db/seeds.rb in the .gitignore file